src/providers/postgres/qgspostgresconn.h

```diff
@@ -183,6 +183,12 @@ class QgsPostgresConn : public QObject
     Q_OBJECT
 
   public:
+    /// @param shared allow using a shared connection. Should never be
```

Comment:
Just guessing, you wanted to say "deprecation warning" and not "deprecation waiting"...
Or I'm confused :)